### PR TITLE
Update zkp.md

### PR DESCRIPTION
**Pull request by szy980525**
I have

# Preamble

For Pull Request initiator, please read the following steps carefully. 请合并请求的发起人仔细阅读下列步骤。

---

If you are creating a `Pull Request` about listed topic below, please go the the `Preview` tab and click the relative link:
如果您要创建一个有关下列内容的的合并请求，请切换到 `Preview` 面板并点击下方相关的链接:

* Select [Awesome-PETs](?expand=1&template=pull_request_template_pets.md) for files under docs/awesome-pets

---

Select a proper `Pull Request` template can accelarate the review process, thanks for your cooperation.
选择一个正确的`合并请求`模版可以加速审查流程，谢谢您的配合

If your `Pull Request` topic is not amoung the list above, please just delete all prepopulated template content.
如果您的`合并请求`内容不在上述的列表里，请删除提前填入的模版内容。

---

Thanks for your contribution to SecretFlow project
感谢您对于 SecretFlow 项目的贡献

### CODE REVIEW

**Comment by szy980525** [comment]
I have read the CLA Document and I ready sign the CLA

**Comment by szy980525** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by xfap** [comment]
Hello, thanks for your contributions! Please obey the citation format(as circled in below picture)!
image

File: docs/awesome-pets/papers/tools/zkp.md [hunk 1]
```diff
@@ -557,3 +557,5 @@ Here just list several interesting applicaitons.
 - Efficient Lifting for Shorter Zero-Knowledge Proofs and Post-Quantum Signatures
   *Daniel Kales, and Greg Zaverucha*
   eprint 2022, [paper](https://eprint.iacr.org/2022/588.pdf), KZ22
+  
+  Public-Coin Zero-Knowledge Arguments with (almost) Minimal Time and Space Overheads
```
Comment (szy980525) [inline docs/awesome-pets/papers/tools/zkp.md:561]: - Public-Coin Zero-Knowledge Arguments with (almost) Minimal Time and Space Overheads
- Alexander R. Block, Justin Holmgren2, Alon Rosen3, Ron D. Rothblum4, and Pratik Soni5
eprint 2020，[paper](https://eprint.iacr.org/2020/1425.pdf)

Comment (xfap) [reply docs/awesome-pets/papers/tools/zkp.md:561]: Hello, you just put your modification in the `conversation` part, you should submit your change and then it'll show in this area(circled in below picture):
image


Besides, you put this paper under the catogory `### Signature from ZKP`, we don't think it's a proper position for this paper. Could you find a more proper location from belows to put it.
image

And the paper's markdown form should be 
```markdown
- Public-Coin Zero-Knowledge Arguments with (almost) Minimal Time and Space Overheads  
  Alexander R. Block, Justin Holmgren, Alon Rosen, Ron D. Rothblum, and Pratik Soni  
  TCC20, [paper](https://eprint.iacr.org/2020/1425.pdf), BHR+20
```

**Comment by xfap** [comment]
@szy980525 Hi, cause long-time couldn't receive your reply, we would close this PR after 24 hours. If you have time for this PR, we welcome you update it according to our request or create a new PR after.

**Pull request closed without merging**